target/linux/ath79/image/generic.mk

```diff
@@ -250,6 +263,11 @@ define Device/compex_wpj531
   IMAGE_SIZE := 16128k
   DEVICE_TITLE := Compex WPJ531 (16MB flash)
   BOARDNAME := WPJ531
+  IMAGES += cpximg-7A03.img cpximg-7A04.img cpximg-7A06.img cpximg-7A07.img
+  IMAGE/cpximg-7A03.bin = $$(IMAGE/sysupgrade.bin) | mkmylofw 0x68a 2
+  IMAGE/cpximg-7A04.bin = $$(IMAGE/sysupgrade.bin) | mkmylofw 0x693 3
+  IMAGE/cpximg-7A06.bin = $$(IMAGE/sysupgrade.bin) | mkmylofw 0x693 3
+  IMAGE/cpximg-7A07.bin = $$(IMAGE/sysupgrade.bin) | mkmylofw 0x693 3
```

Comment:
Yes, that's the idea.
One should discuss how those "factory" images are named.
It might be desirable to stick to the "factory" term, so use "factory-7A03.bin" instead.
However, there is also "eva" for FritzBox etc., so I really have no definite opinion on this one so far.

Reply:
the image used when installing the firmware with the u-boot prompt (tftp, erase, cp.b - like in the wiki) is identical to the sysupdate file.
however, this cpximg can not be used for that process. it uses the cpximg command.

Reply:
The parameters for the last three lines are the same. Mistake?